Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/ScheduleDatabase.java

```diff
@@ -55,7 +54,9 @@
         public String speakerName;
         public String talkTitle;
         public String photo;
-        public String time;
+        public ZonedDateTime localStartTime;
```

Comment:
personally would add comment say used for sorting (otherwise would expect a localEndTime field as well)